lib/chef-acceptance/chef_runner.rb

```diff
@@ -33,7 +41,13 @@ def run!
       )
 
       Bundler.with_clean_env do
-        chef_shellout.run_command
+        begin
+          chef_shellout.run_command
+        rescue Mixlib::ShellOut::CommandTimeout => e
+          suite_logger.log("Command timed out after #{app_options.timeout}s")
```

Comment:
Missing space between `timeout` and `"s"`. Also would be good to spell out it it is msecs or secs.